### ISSUE TITLE
[Bug Report] segementation does not work 

### ISSUE THREAD

**Issue by samibouziri** [NVIDIA-Omniverse/orbit#225]
### Describe the bug

For the Camera Sensor initialized with CameraCfg the instance segementation does not work and throw this error:
``` 
2024-01-30 22:43:38 [8,436ms] [Error] [carb.events.python] SyntheticDataException: invalid node attribute /Render/PostProcess/SDGPipeline/RenderProduct_Replicator_instance_segmentation.inputs:semanticTypes

At:
  /home/sbouziri/.local/share/ov/pkg/isaac_sim-2023.1.0-hotfix.1/kit/exts/omni.syntheticdata/omni/syntheticdata/scripts/SyntheticData.py(1330): _set_node_attributes
  /home/sbouziri/.local/share/ov/pkg/isaac_sim-2023.1.0-hotfix.1/kit/exts/omni.syntheticdata/omni/syntheticdata/scripts/SyntheticData.py(1449): activate_node_template
  /home/sbouziri/.local/share/ov/pkg/isaac_sim-2023.1.0-hotfix.1/extscache/omni.replicator.core-1.10.10+105.1.lx64.r.cp310/omni/replicator/core/scripts/annotators.py(562): attach
  /home/sbouziri/Orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/sensors/camera/camera.py(402): _initialize_impl
  /home/sbouziri/Orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/sensors/sensor_base.py(262): _initialize_callback
  /home/sbouziri/Orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/sensors/sensor_base.py(63): <lambda>
  /home/sbouziri/Orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/sim/simulation_context.py(423): render
  /home/sbouziri/.local/share/ov/pkg/isaac_sim-2023.1.0-hotfix.1/exts/omni.isaac.core/omni/isaac/core/simulation_context/simulation_context.py(561): play
  /home/sbouziri/.local/share/ov/pkg/isaac_sim-2023.1.0-hotfix.1/exts/omni.isaac.core/omni/isaac/core/simulation_context/simulation_context.py(387): initialize_physics
  /home/sbouziri/.local/share/ov/pkg/isaac_sim-2023.1.0-hotfix.1/exts/omni.isaac.core/omni/isaac/core/simulation_context/simulation_context.py(408): reset
  /home/sbouziri/Orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/sim/simulation_context.py(352): reset
  /home/sbouziri/sami_bouziri_345/src/orbit/src/test_env.py(213): main
  /home/sbouziri/sami_bouziri_345/src/orbit/src/test_env.py(224): <module>

~*~*~*
~*~*~* Direct GPU Helper:
~*~*~*   6 rigid dynamics
~*~*~*   3 articulations, maxLinks=11
~*~*~*
2024-01-30 22:43:38 [8,806ms] [Warning] [omni.usd] Coding Error: in _getAttributeValueWithPrecision at line 39 of ../../../source/extensions/omni.hydra.scene_delegate/plugins/omni.hydra.scene_delegate.plugin/OmniHydraXformOpCacheItem.cpp -- Failed verification: ' attr.Get<DoubleType_>(&valued, timeCode) ' -- Failed to get value from /__Prototype_23/mesh.xformOp:scale at DEFAULT

2024-01-30 22:43:38 [8,806ms] [Warning] [omni.hydra.scene_delegate.plugin] Calling getBypassRenderSkelMeshProcessing for prim /World/envs/env_0/Robot/sgr532_link6/visuals.proto_mesh_0_id0 that has not been populated
2024-01-30 22:43:39 [9,018ms] [Warning] [omni.syntheticdata.plugin] OgnSdPostInstanceMapping missing source aov InstanceIdTokenMapSD
2024-01-30 22:43:39 [9,018ms] [Warning] [omni.replicator.core.plugin] InstanceSegmentation : missing RenderVar InstanceMapSD
2024-01-30 22:43:39 [9,053ms] [Warning] [omni.syntheticdata.plugin] SdRenderVarToRawArray missing valid input renderVar BoundingBox2DTightSD
2024-01-30 22:43:39 [9,053ms] [Warning] [omni.syntheticdata.plugin] OgnSdInstanceMapping missing valid input renderVar InstanceMappingInfoSDhost
[INFO]: Setup complete...
2024-01-30 22:43:39 [9,098ms] [Error] [__main__] 'NoneType' object does not support item assignment
2024-01-30 22:43:39 [9,098ms] [Error] [__main__] Traceback (most recent call last):
  File "/home/sbouziri/sami_bouziri_345/src/orbit/src/test_env.py", line 224, in <module>
    main()
  File "/home/sbouziri/sami_bouziri_345/src/orbit/src/test_env.py", line 218, in main
    run_simulator(sim, scene)
  File "/home/sbouziri/sami_bouziri_345/src/orbit/src/test_env.py", line 157, in run_simulator
    scene.reset()
  File "/home/sbouziri/Orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/scene/interactive_scene.py", line 245, in reset
    sensor.reset(env_ids)
  File "/home/sbouziri/Orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/sensors/camera/camera.py", line 321, in reset
    self._update_poses(env_ids)
  File "/home/sbouziri/Orbit/source/extensions/omni.isaac.orbit/omni/isaac/orbit/sensors/camera/camera.py", line 513, in _update_poses
    self._data.pos_w[env_ids] = poses
TypeError: 'NoneType' object does not support item assignment
```

Semantic segmentation on the other hand works partially. It returns all objects as 1 when no objects have semantic tags and returns ints (as supposed) when semantic tags are added. the issue is that we don t have a semantic_id matching with the class meaning that the segmentation is useless.
Finally Semantic segmentation + Colorized does not work. it returns a tensor of size (num_envs, h, w ) instead of (num_envs, h, w, 4) 

If you think this is not a bug can you provide an example code on how to do segmentation. It is not available in the documentation neither the code base

### System Info

Describe the characteristic of your environment:

- commit: 620ce2b08dc905d0a2b46e986b9da6adbe1f4637
- Isaac Sim Version: 2023.1.0-rc.42+2023.1.667.d641947f.tc
- OS:  ubuntu-22.04.3
- GPU: GeForce RTX 3090 VISION OC 24G GDDR6X
- CUDA: 12.2
- GPU Driver: 535.113.01

### Checklist

- [x] I have checked that there is no similar issue in the repo (**required**)
- [x] I have checked that the issue is not in running Isaac Sim itself and is related to the repo

**Comment by Mayankm96** [comment]
Thanks for bring this up. @pascal-roth can you please take a look?

**Comment by samibouziri** [comment]
Small update. I found the segmentation label mapping in scene['camera'].data.info. That said colorized does not work well as explained above. in the info data we find a mapping between colors and class types but in the output['sematic_segmentation] there is no colors.
Instance segmentation does not work as mentioned above.

**Comment by Mayankm96** [comment]
Thank you pointing this out. Release upgrade to Isaac Sim 2023.1 broke the sensor for semantic types. It is fixed now with [8dea21a](https://github.com/isaac-orbit/orbit/commit/8dea21a8ac805ab4dd804fc9927110f3c27f8b3b).
